### PR TITLE
(DOCSP-8163) and (DOCSP-12999) Add new default limits to the system variables table, and clean up the table

### PR DESCRIPTION
**Pull request by JuliaMongo**
This PR addresses two tickets in the BI Connector documentation:

1. [DOCSP-8163](https://jira.mongodb.org/browse/DOCSP-8163) -- asking to add missing variables and their defaults.
2. [DOC-12229](https://jira.mongodb.org/browse/DOCSP-12999) -- asking to streamline the table by removing its fifth column (no pun intended). As part of this ticket, also removed the currently not used /includes/fact-system-vars.rst.
The [build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5fa1a7428f1fd0fda5cce246) succeeds without warnings related to this change. 
[Content on staging](https://docs-mongodbcom-staging.corp.mongodb.com/02d7bd1/bi-connector/docsworker-xlarge/DOCSP-8163/reference/system-variables).

Changes made:
1. Added missing variables.
2. Removed fifth column in the table and adjusted the value for log_level to account for cloud and on prem.
3. Removed the unnecessary file from includes.
NOTE:
In DOCSP-8163, one new variable is added and then in the same comment it is marked as no-op. Thus, I haven't included it.

Thank you for your review.

### CODE REVIEW

**Review by steveren — APPROVED** [review]
LGTM
Nicely done!

**Comment by JuliaMongo** [comment]
@rychipman hi Ryan, I need a tech review for this doc ticket. Devin has left the team and so she suggested I tag you and ask a review from Patrick M. I did ask Patrick, but git's UI does not let me select Patrick from the drop down as a reviewer. It must be related to access to project, etc. So, I guess we are all set, unless there is an easy way to fix this. Sending this note as an FYI only about this ticket. I tagged Patrick in Jira and asked for a review. Thanks!

File: source/reference/system-variables.txt [hunk 1]
```diff
@@ -43,27 +41,38 @@ Overview
...
    * - ``max_nested_table_depth``
      - integer
      - Specifies the maximum number of unique nested field paths that
        ``mongosqld`` maps to a relational table for a collection.
      - ``50``
-     - ``50``
-   * - ``max_num_columns_per_table``
+   * - ``max_num_global_tables``
+     - integer
+     - The maximum number of tables allowed in a table with an automatically-
```
Comment (pmeredit) [inline source/reference/system-variables.txt:52]:
```suggestion
     - The maximum number of tables allowed in a mongodb deployment with an automatically-
```

File: source/reference/system-variables.txt [hunk 2]
```diff
@@ -43,27 +41,38 @@ Overview
...
+       generated schema.
+     - ``4000``
+   * - ``max_num_tables_per_collection``
+     - integer
+     - The maximum number of tables allowed in a collection in a table with an
+       automatically-generated schema.
+     - ``200``
+   * - ``max_nested_table_depth``
+     - integer
+     - The default limit for table's nesting depth in a table with an
```
Comment (pmeredit) [inline source/reference/system-variables.txt:62]:
```suggestion
     - The default limit for table's nesting depth, that is the depth of arrays that will be translated to tables, in a collection with an
```

File: source/reference/system-variables.txt [hunk 3]
```diff
@@ -91,7 +100,18 @@ Overview
...
      -  ``off``
-     -  ``off``
+   * - ``reconcile_arithmetic_agg_functions``
+     - boolean
+     - Allows you to choose the behavior where MongoDB does not
+       automatically convert types for some queries using arithmetic
+       aggregate functions. In MongoDB version 3.6 and earlier, this
+       behavior results in improved performance, compared with implicit
+       type conversions for arithmetic aggregation functions in subsequent
+       versions.
```
Comment (pmeredit) [inline source/reference/system-variables.txt:110]: Excellent!

**Review by pmeredit — CHANGES_REQUESTED** [review]
Just some minor changes. I approve once these are added. Great job!

File: source/reference/system-variables.txt (hunk 1 above)
Comment (JuliaMongo) [reply source/reference/system-variables.txt:52]: Done

File: source/reference/system-variables.txt (hunk 2 above)
Comment (JuliaMongo) [reply source/reference/system-variables.txt:62]: Updated to become: "The default limit for table's nesting depth of arrays
       that MongoDB translates to tables, in a collection with an 
       automatically-generated schema."

**Comment by JuliaMongo** [comment]
Included reviews from Patrick and Steve (thank you), and merging.

**Pull request merged**